v2.0/begin-transaction.md

```diff
@@ -27,16 +27,15 @@ In CockroachDB, the following are aliases for the `BEGIN` statement:
 
 The following aliases also exist for [isolation levels](transactions.html#isolation-levels):
 
-- `REPEATABLE READ` is an alias for `SERIALIZABLE`
-- `READ UNCOMMITTED` and `READ COMMITTED` are aliases for `SNAPSHOT`
+- `READ UNCOMMITTED`, `READ COMMITTED`, and `REPEATABLE READ` are aliases for `SERIALIZABLE`
 
 For more information on isolation level aliases, see [Comparison to ANSI SQL Isolation Levels](transactions.html#comparison-to-ansi-sql-isolation-levels).
 
 ## Parameters
 
 | Parameter | Description |
 |-----------|-------------|
-| `ISOLATION LEVEL` | If you don't want the transaction to run as `SERIALIZABLE` (CockroachDB's default, which provides the highest level of isolation), you can specify `SNAPSHOT`, which can provide better performance in high-contention scenarios.<br/><br/>For more information, see [Transactions: Isolation Levels](transactions.html#isolation-levels).<br/><br/>**Default**: `SERIALIZABLE` |
+| `ISOLATION LEVEL` | If you don't want the transaction to run as `SERIALIZABLE` (CockroachDB's default, which provides the highest level of isolation), you can specify `SNAPSHOT` (**deprecated**), which can provide fewer restarts in high-contention scenarios, but can cause inconsistencies.<br/><br/>For more information, see [Transactions: Isolation Levels](transactions.html#isolation-levels).<br/><br/>**Default**: `SERIALIZABLE` |
```

Comment:
The description of `SNAPSHOT` in the transactions doc is much more direct.  Should we follow suit here, for example:

> By default, transactions in CockroachDB implement the strongest ISO isolation level: `SERIALIZABLE`. At this isolation level, transactions will never result in anomalies. The `SNAPSHOT` isolation level is still supported as well for backwards compatibility, but you should avoid using it. It provides little benefit in terms of performance and can result in inconsistent state under certain complex workloads.
>
> For more information, see [Transactions: Isolation Levels](transactions.html#isolation-levels).
>
>**Default**: `SERIALIZABLE`

Reply:
Done.